Ghost CMS migrator, more updates (#139)
Automattic/newspack-migration-tools: docs/GhostCMS.md

* `GhostCMSMigrator_cmd_ghostcms_import.log` - This log file will list all content that was imported along with any warning or errors encountered.
* `GhostCMSMigrator_cmd_ghostcms_import.log-skips.log` - If a post was already imported, it will not be imported again. A list of "skipped" posts will be written to this file.

### Step 7: Download files from external host

The content has now been imported, but the images are still hosted on the external host. You will need to download the images to your local WordPress installation.

> Recommended to use the [Newspack Post Image Downloader](https://github.com/Automattic/newspack-post-image-downloader) plugin to download the images. Follow the [README.md](https://github.com/Automattic/newspack-post-image-downloader)) file in plugin repo for usage and best workflow.

Reviewer: ```suggestion
Recommended to use the [Newspack Post Image Downloader](https://github.com/Automattic/newspack-post-image-downloader) plugin to download the images. Follow the [README.md](https://github.com/Automattic/newspack-post-image-downloader) file in plugin repo for usage and best workflow.
```

* `GhostCMSMigrator_cmd_ghostcms_import.log` - This log file will list all content that was imported along with any warning or errors encountered.
* `GhostCMSMigrator_cmd_ghostcms_import.log-skips.log` - If a post was already imported, it will not be imported again. A list of "skipped" posts will be written to this file.

### Step 7: Download files from external host

> The content has now been imported, but the images are still hosted on the external host. You will need to download the images to your local WordPress installation. Recommended to use the [Newspack Post Image Downloader](https://wordpress.org/plugins/newspack-post-image-downloader/) plugin to download the images.

Reviewer: You may want to specific that it's the "in-content" images are still hosted on the external host, since the featured images and avatars were fetched already.

> The content has now been imported, but the images are still hosted on the external host. You will need to download the images to your local WordPress installation. Recommended to use the [Newspack Post Image Downloader](https://wordpress.org/plugins/newspack-post-image-downloader/) plugin to download the images.

Reviewer: See "suggested change":

* You may want to specify that it's the "in-content" images are still hosted on the external host, since the featured images and avatars were fetched already.
* Broken link.

```suggestion
The content has now been imported, but the in-content images are still hosted on the external host. You will need to download the images to your local WordPress installation. Recommended to use the [Newspack Post Image Downloader](https://github.com/Automattic/newspack-post-image-downloader) plugin to download the images.
```

```
wp plugin install https://github.com/Automattic/newspack-post-image-downloader/releases/latest/download/newspack-post-image-downloader.zip --activate
```

> Here is the recommended workflow:

Reviewer: It's nice to have a shortened workflow here, but it also seem redundant since the steps are also in the post image downloader [readme](https://github.com/Automattic/newspack-post-image-downloader).  I don't have a strong opinion either way, but I'd recommend this:

```suggestion
Here is the recommended workflow (all available options [here](https://github.com/Automattic/newspack-post-image-downloader)):
```
